### DISCUSSION TITLE
vertical taskbar suggestions

### DISCUSSION THREAD

**Discussion by gmit3** [valinet/ExplorerPatcher#368] [Ideas]
Hi,

I use a vertical taskbar. I was wondering, would it be possible to do the following:
 - reposition Keyboard language and Control Center icons to the same row as the Notification icon, to save two vertical rows
 - when tabs don't fit in the taskbar, a scrollbar is shown. The experience would be much better if scrollbar would react to the wheel movement while mouse pointer is above taskbar.

Tnx!

**Comment by valinet** [comment]
Interesting ideas. Thanks for the suggestions.

**Reply by 3N4N** [reply]
@valinet Thanks for the product.  I would like 2 rows on vertical taskbar as well.  It seems that's only available on horizontal taskbar.
